src/app/GitUI/Editor/Diff/DiffHighlightService.cs

```diff
@@ -368,43 +391,6 @@ private static (int LengthIdenticalAtStart, int LengthIdenticalAtEnd) AddDiffere
         return (lengthIdenticalAtStart, lengthIdenticalAtEnd);
     }
 
-    /// <summary>
-    ///  Matches related removed and added lines in a consecutive block of a patch document and marks identical parts dimmed.
-    /// </summary>
-    private void MarkInlineDifferences(IDocument document)
-    {
-        int line = 0;
-        bool found = false;
-
-        // Skip the first pair of removed / added lines, which uses to contain the filenames - but without highlighting
-        _ = GetRemovedLines(document, ref line, ref found);
-        if (found)
-        {
-            _ = GetAddedLines(document, ref line, ref found);
-        }
```

Comment:
How / does the DiffLineNumAnalyzer exclude such lines?

Reply:
DiffLineType.Minus/DiffLineType.Plus is only set when a @@ header has been found